--- protoc-gen-lua-0_0.orig/CMakeLists.txt	Wed Dec 31 16:00:00 1969
+++ protoc-gen-lua-0_0.orig/CMakeLists.txt	Fri Aug  2 09:26:42 2013
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
+project(pb C)
+
+set(PB_SOURCE
+protobuf/pb.c
+)
+
+add_library(pb STATIC ${PB_SOURCE})
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O2 -Wall -fPIC")
+
+install(TARGETS pb DESTINATION lib)
+
+

